src/wtc-gallery-component.js

```diff
@@ -95,8 +94,8 @@ class Gallery extends ElementController {
       this.prevBtn = document.createElement("button");
       this.prevBtn.innerHTML = this.options.prevBtnMarkup;
 
-      _u.addClass("gallery__nav gallery__nav-next", this.nextBtn);
-      _u.addClass("gallery__nav gallery__nav-prev", this.prevBtn);
+      this.nextBtn.className = "gallery__nav gallery__nav-next";
```

Comment:
Any reason not to use classList here? @andrewrubin

Reply:
Yes! Unfortunately `classList` only accepts one string with no spaces per call. Since these elements are brand new and don't have any classes to begin with, I opted to set their `className` method instead of two calls each to `.classList`.

Reply:
You can pass multiple class names: https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/add
But this is okay, as you mentioned these are new elements.